src/components/AddTask.tsx

```diff
@@ -0,0 +1,79 @@
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+import { getFormattedDate } from "../utils/dateFunctions";
+import circledPlusIcon from '../assets/icons/circledPlusIcon.png'
+import { AddTaskProp } from "../types/types";
+
+export function AddTask(props: AddTaskProp) {
+
+    return (
+        <>
+            <div className={props.isModalOpen ? "add-task-container" : "add-task-container zIndex"} data-testid="container">
```

Comment:
`data-testid="container` - what is this `data-testid` for?

Reply:
it is for testing purpose Anna

Reply:
are you testing anything here?

Reply:
https://github.com/PreethiKalyani99/Clockify-Typescript/commit/5c00cee71f874b0e45bb06ddc7cafcc1257d3085

Removed data-testid Anna